scripts/incident_schema_v3.json

```diff
@@ -3,7 +3,7 @@
     "title": "",
     "plural_title": "",
     "definitions": {
-        "photo": {
+        "photo_zu3df": {
```

Comment:
Shouldn't these be prepended with 'driver' to match what the generator does now?